### PR TITLE
fix:add color white on the span itself

### PR DESCRIPTION
**Pull request by itaykinnrot**
add color to span

### CODE REVIEW

File: src/components/unmute-indication/_unmute-indication.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+$color: 'white';
```
Comment (dan-ziv) [inline src/components/unmute-indication/_unmute-indication.scss]: no need to define new scss var for white, this is already defined in the _variables.scss

File: src/components/unmute-indication/_unmute-indication.scss [hunk 2]
```diff
@@ -6,7 +8,7 @@
   z-index: 15;
 
   a {
-    color: white;
+    color: $color;
```
Comment (dan-ziv) [inline src/components/unmute-indication/_unmute-indication.scss]: `color: $white;`

File: src/components/unmute-indication/_unmute-indication.scss [hunk 3]
```diff
@@ -34,6 +36,7 @@
     opacity: 1;
     transition: transform 100ms, opacity 100ms;
     display: inline-block;
+    color: $color;
```
Comment (dan-ziv) [inline src/components/unmute-indication/_unmute-indication.scss]: `color: $white;`

File: src/components/unmute-indication/_unmute-indication.scss (hunk 1 above)
Comment (itaykinnrot) [reply src/components/unmute-indication/_unmute-indication.scss]: its more for the color of the unmute button

Comment (dan-ziv) [reply src/components/unmute-indication/_unmute-indication.scss]: the color is white, so until this will be changed, no need for another definition IMO

Comment (itaykinnrot) [reply src/components/unmute-indication/_unmute-indication.scss]: @OrenMe ?

Comment (OrenMe) [reply src/components/unmute-indication/_unmute-indication.scss]: @dan-ziv as it is used in multiple places and we will need to support theming it's better to start putting such things as settings attribute - although it is not connected to anything ATM.
Also - setting a css param called white and then using it instead has no value over inlining it.

Comment (OrenMe) [reply src/components/unmute-indication/_unmute-indication.scss]: @dan-ziv if there's no other issue please remove the change request. thanks.

**Pull request merged**